### PR TITLE
Clean up changes on OB/C pages

### PR DESCRIPTION
**Pull request by bipuladh**
- Added resource link to storage class
 - Changed helper functions due to changes in CRD

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/utils.ts [hunk 1]
```diff
@@ -19,16 +19,14 @@ export type PrometheusMetricResult = {
 };
 
 export const getOBCPhase = (obc: K8sResourceKind): string => {
-  const phase: string = _.get(obc, 'status.Phase');
-  return phase ? phase.charAt(0).toUpperCase() + phase.substring(1) : 'Lost';
+  return _.get(obc, 'status.phase', 'Lost');
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/utils.ts]:
```suggestion
  return _.get(obj, 'status.phase', 'Lost');
```
generalize the functions `getOBCPhase` and `getOBPhase`

File: frontend/packages/noobaa-storage-plugin/src/utils.ts [hunk 2]
```diff
@@ -19,16 +19,14 @@ export type PrometheusMetricResult = {
 };
 
 export const getOBCPhase = (obc: K8sResourceKind): string => {
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/utils.ts]:
```suggestion
export const getPhase = (obj: K8sResourceKind): string => {
```

File: frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx [hunk 3]
```diff
@@ -110,7 +110,7 @@ const AttachDeploymentToOBCModal = withHandlePromise((props: AttachDeploymentToO
     e.preventDefault();
     handlePromise(k8sPatch(DeploymentModel, requestDeployment, getPatches()))
       .then((res) => {
-        history.push(resourceObjPath(res, referenceFor(res)));
+        history.push(`${resourceObjPath(res, referenceFor(res))}/environment`);
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx]: Why this change ?

File: frontend/packages/noobaa-storage-plugin/src/utils.ts (hunk 1 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/src/utils.ts]: ack

File: frontend/packages/noobaa-storage-plugin/src/utils.ts (hunk 2 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/src/utils.ts]: ack

File: frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx (hunk 3 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx]: This is where the user can see the actual changes made by the attachment. We are adding env variables.
